pkg/types/powervs/machinepools.go

```diff
@@ -0,0 +1,55 @@
+package powervs
+
+// +kubebuilder:validation:Enum=capped;dedicated;shared
```

Comment:
An empty string is a valid value, too.
```suggestion
// +kubebuilder:validation:Enum="";capped;dedicated;shared
```